src/test/integration/channel_creation_duplicate_name_spec.js

```diff
@@ -1,30 +1,55 @@
 require('./integration_config.js');
 
 var channelName = utils.randomChannelName();
-var jsonBody = JSON.stringify({ "name": channelName});
-var channelResource = channelUrl + "/" + channelName;
-var testName = "channel_creation_duplicate_name_spec";
+var channelResource = channelUrl + '/' + channelName;
 
-utils.configureFrisby();
+describe(__filename, function () {
 
-frisby.create(testName + ': Making sure channel resource does not yet exist.')
-    .get(channelResource)
-    .expectStatus(404)
```

Comment:
Yeah, I saw a lot of things I didn't quite agree with but I wanted to focus on recreating what Frisby was doing. I'm going to make a pass on cleaning up unnecessary steps.